DirectEmployers/MyJobs
# Fixed "All categories selecting when editing wizard" + Refactored some stuff

## What's this PR do?
* Fixes the bug in the title of this PR
* Refactored some of `AddContact.js` component

## Screenshots (if appropriate)

## Where should the reviewer start?
`AddContact.js`, `cboWizardActions.js`, `cboWizardReducer.js`

## How should this be manually tested?
1. Employers -> TJD
2. +New Organization
3. Fill out required fields
4. Select a category or two
5. Click the **Next** button
6. Entered Req'd data & click **Next**
7. Click **Edit Contact Details**
8. Scroll down to the categories
9. Only the ones originally selected should be checked.
